src/components/pages/dashboard/panels/_examplePanel/examplePanel.scss

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.example-panel-container {
```

Comment:
The panel-container class already has these properties by default. Should we include both classes in the example panel so that the css isn't needed here? Or is the idea to have a guide on how to customize the display more?

Reply:
When they add real stuff to their panel, they'll need a place to put it.  That's really why I created this file.